zha/zigbee/device.py

```diff
@@ -243,6 +243,10 @@ def __init__(
             if ep_id != 0:
                 self._endpoints[ep_id] = Endpoint.new(endpoint, self)
 
+    def __repr__(self) -> str:
+        """Return a string representation of the device."""
+        return f"{self._zigpy_device.__repr__()} - quirk_applied: {self.quirk_applied} - quirk_or_device_class: {self.quirk_class} - quirk_id: {self.quirk_id}"
```

Comment:
You could argue this line should be split 😄

Reply:
That’s fine. How do you want it split?

Reply:
Just code-wise, so it's not that long. Maybe something like this?
```suggestion
        return (
            f"{repr(self._zigpy_device)} - "
            f"quirk_applied: {self.quirk_applied} - "
            f"quirk_or_device_class: {self.quirk_class} - "
            f"quirk_id: {self.quirk_id}"
        )
```